pkg/errors/request.go

```diff
@@ -27,11 +27,11 @@ import (
 )
 
 // ValidateUnaryContext validates that a context for a unary request is valid
-// and contains all required information.
+// and contains all required information, and returns a YARPC error with code
+// yarpcerrors.CodeInvalidArgument otherwise.
 func ValidateUnaryContext(ctx context.Context) error {
```

Comment:
It's a catchall for error helper functions, ie error constructors, see previous comments.